pkg/config/config.go

```diff
@@ -42,9 +42,9 @@ type (
 	}
 
 	CloudResourceLink struct {
-		Source string `json:"source,omitempty" yaml:"source,omitempty" toml:"source,omitempty"`
-		Target string `json:"target,omitempty" yaml:"target,omitempty" toml:"target,omitempty"`
-		Type   string `json:"type,omitempty" yaml:"type,omitempty" toml:"type,omitempty"`
+		Source core.ResourceId `json:"source,omitempty" yaml:"source,omitempty" toml:"source,omitempty"`
+		Target core.ResourceId `json:"target,omitempty" yaml:"target,omitempty" toml:"target,omitempty"`
+		Type   string          `json:"type,omitempty" yaml:"type,omitempty" toml:"type,omitempty"`
```

Comment:
i wonder if we should delete this, doesnt seem like we are going to use it, doesnt have to be done in this pr, just saw it